### PR TITLE
README.md changes(WIP)

### PR DESCRIPTION
**Pull request by hm90121**
- added table of contents
- corrected typos and added links
- added zus overview
still a work in progress

### CODE REVIEW

**Review by boddumanohar — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,24 +1,44 @@
...
-1. ZS3Server is the main component which will communicate directly with 0chain allocation 
+1. ZS3Server is the main component that will communicate directly with Züs storage.
 
-2. [LogSereach](/logsearchapi/README.md) API is the log component which will store the audit log from S3 server and It will be consumed using ZS3 API
+2. [LogSearch](/logsearchapi/README.md) API is the log component that will store the audit log from the S3 server and it will be consumed using ZS3 API
 
 3. [MinioClient](/client-api/README.md) is the component that will communicate directly to the zs3server and it is protected using access and secret key. 
 
+## Building zs3-server
+Prerequisites to run MinIO ZCN gateway: [wallet.json can be created using zwalletcli](https://github.com/0chain/zwalletcli#creating-and-restoring-wallets), [config.yaml](https://github.com/0chain/zboxcli/blob/staging/network/config.yaml), and [allocation.txt can be created using zboxcli](https://github.com/0chain/zboxcli/tree/staging#create-new-allocation).
```
Comment (Kishan-Dhakan) [inline README.md]: Please write these as different points/steps rather than one comma separated sentence.

File: README.md [hunk 2]
```diff
@@ -29,32 +49,24 @@ export MINIO_ROOT_PASSWORD=someminiopassword
...
-
 > If you want to debug on local you might want to build with `-gcflags="all=-N -l"` flag to view all the objects during debugging.
 
-## Run using docker 
-
-To build and run minio sevrer component in any machine you will need first to install docker and docker-compose 
+## Running zs3-server 
 
-1. Make sure docker and docker-compose in your machine
+To build and run minio server component you need to first install [docker](https://www.docker.com/products/docker-desktop/).
```
Comment (Kishan-Dhakan) [inline README.md]: To build and run `the`

File: README.md [hunk 3]
```diff
@@ -29,32 +49,24 @@ export MINIO_ROOT_PASSWORD=someminiopassword
...
+## Running zs3-server 
 
-1. Make sure docker and docker-compose in your machine
+To build and run minio server component you need to first install [docker](https://www.docker.com/products/docker-desktop/). 
 
-2. Make sure you have the allocation ready in the default folder ``~/.zcn``
+1. Make sure you have the allocation ready in the default folder ``~/.zcn``
 
-3. Run docker-compose command like the following
+3. Run the docker-compose command inside the zs3server directory.
```
Comment (Kishan-Dhakan) [inline README.md]: Change 3. to 2.

File: README.md (hunk 2 above)
Comment (Kishan-Dhakan) [reply README.md]: remove the `first`

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

**Review by stewartie4 — COMMENTED** [review]
Happy to merge minus Kishan's comments

**Comment by hm90121** [comment]
@Kishan-Dhakan i have included your suggestions

**Review by Kishan-Dhakan — APPROVED** [review]

**Pull request merged**